app/tasks/mailchimp.py

```diff
@@ -108,10 +109,21 @@ def send_new_briefs_email():
         current_app.logger.error("An Mailchimp API error occurred, aborting: %s %s", e, e.response)
         raise e
 
-    # send the campaign
+    # schedule the campaign to send at least an hour from runtime, rounded up to the nearest 15 minute mark
     try:
-        client.campaigns.actions.send(campaign_id=campaign['id'])
-        current_app.logger.info('Sent new briefs seller email campaign')
+        schedule_time = pendulum.now('UTC').add(hours=1)
+        current_minute = schedule_time.minute
+        if current_minute % 15 != 0:
```

Comment:
why do we care about round to nearest 15 mins?

Reply:
It's a requirement of the Mailchimp API - it has to be scheduled on a 15 minute mark:

http://developer.mailchimp.com/documentation/mailchimp/reference/campaigns/#action-post_campaigns_campaign_id_actions_schedule

Reply:
Ok. Tried to see if there was an easier way with Pendulum but couldn't.